src/main/java/jp/skypencil/errorprone/slf4j/DoNotUseStaticSlf4jLogger.java

```diff
@@ -0,0 +1,59 @@
+package jp.skypencil.errorprone.slf4j;
+
+import static com.google.errorprone.BugPattern.SeverityLevel.SUGGESTION;
+import static com.google.errorprone.matchers.Matchers.allOf;
+import static com.google.errorprone.matchers.Matchers.isField;
+import static com.google.errorprone.matchers.Matchers.isStatic;
+
+import com.google.auto.service.AutoService;
+import com.google.errorprone.BugPattern;
+import com.google.errorprone.VisitorState;
+import com.google.errorprone.bugpatterns.BugChecker;
+import com.google.errorprone.bugpatterns.BugChecker.VariableTreeMatcher;
+import com.google.errorprone.fixes.Fix;
+import com.google.errorprone.matchers.Description;
+import com.google.errorprone.matchers.Matcher;
+import com.sun.source.tree.ModifiersTree;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Use classes from the Java API instead of Sun classes. ([squid:S1191](https://sonarcloud.io/organizations/kengotoda-github/rules#rule_key=squid%3AS1191))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=jp.skypencil.errorprone.slf4j%3Aerrorprone-slf4j&pullRequest=4&issues=AWRd1zxd1IJZDfsS72jd&open=AWRd1zxd1IJZDfsS72jd)

[ISSUE_KEY]: AWRd1zxd1IJZDfsS72jd